api/exclusions/exclusionService.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Response } from 'express';
+import { EnhancedRequest } from '../lib/models';
+
+export async function getUserExclusions(req: EnhancedRequest, res: Response, next: NextFunction) {
+    const isJudge = req.session.passport.user.userinfo.roles.includes('caseworker-ia-iacjudge');
```

Comment:
This assumes that `req.session.passport.user.userinfo.roles` is always defined, what happens when one of the properties is not defined. Can we have defensive mechanism here.

Reply:
The only reason when this is not populated is when the user is not logged in. So, we do not need to worry that scenario.